cvxportfolio/simulator.py

```diff
@@ -147,10 +148,16 @@ class MarketData:
     
     def __init__(self, 
         universe = [], 
-        returns=None, volumes=None, prices=None, 
-        cash_key='USDOLLAR', base_location=BASE_LOCATION, 
-        periods_per_year=252, min_history=252, max_contiguous_missing='10d', 
-        **kwargs):
+        returns=None,
+        volumes=None,
+        prices=None, 
+        cash_key='USDOLLAR',
+        base_location=BASE_LOCATION, 
+        periods_per_year=PPY,
+        min_history=PPY,
+        max_contiguous_missing='10d',
+        **kwargs,
```

Comment:
I note that these kwargs, as well as the ones in the cost functions, are not currently used.

Reply:
They are, you can override them by passing them to MarketSimulator (it all passes the same `**kwargs`).